exercises/sheet4/marta_bustins_marta_gonzalez_esteve_tarrago/4.tex

```diff
@@ -0,0 +1,186 @@
+\def\A{\mat{A}}
+\def\B{\mat{B}}
+\def\u{\vect{u}}
+\def\f{\vect{f}}
+\def\b{\vect{b}}
+\def\x{\vect{x}}
+
+%_______________________________________________________________________________
+%                                      Caratula
+\begin{document}
+
+\begin{center}{\Large{\sc ASSIGNMENT 4, discrete geometry }}\end{center}
+
+\begin{center}{\textit{December, 10th 2018}}\end{center}
+
+\begin{center}{\sc{Marta Bustins, Esteve Tarragó, Marta González}
+}\end{center}
+
+\bigskip
+
+
+\begin{enumerate}
+\item Definition 9.2 in Ziegler's \emph{Lectures on Polytopes} constructs the linear map
+  \[
+    P
+    \ \xrightarrow{\pi^c}\ 
+    Q^c :=
+    \Big\{ \binom{\pi(x)}{cx} : x\in P\Big\}
+    \ \subset \
+    \RR^{q+1}
+  \]
+  from a projection $\pi:P\subset\RR^p\to Q\subset\RR^q$ and a linear function $c\in(\RR^q)^\star$.
+  Is it possible to give an algorithm to determine the set of lower faces $\mathcal L^\downarrow(Q^c)$ of $Q^c$ from just the set of facet normals of~$Q$, the projection~$\pi$, and the linear function~$c$, without running a convex hull algorithm on $Q^c$?
+  
+  \bigskip
+  It doesn't exist such a deterministic algorithm because for a given input it should always return the same output. In the following examples (rectangle and triangle), the input of the algorithm is the same and the output should be different.
+  
+    \begin{figure}[H]
+    \centering
+    \begin{minipage}{0.45\textwidth}
+        \centering
+			\includegraphics[width=6cm]{A.jpg}
+        %\caption{first figure}%
+    \end{minipage}\hfill
+    \begin{minipage}{0.45\textwidth}
+        \centering
+			\includegraphics[width=6cm]{B.jpg}
+        %\caption{second figure}%
+    \end{minipage}
+\end{figure}
+
+The input of the algorithm is the set of facet normals of $Q$, the projection $\pi$ (which is removing the second coordinate) and the liniear function c:=(1,0). And the lower faces of $Q^c$ are different in each case.
+
+  \bigskip\bigskip
+\item Show that
+  \[
+    \int_P f(x)\,\text{d}x
+    \ = \
+    vol(P) \cdot f(p_0)
+  \]
+  for any polytope $P$ and linear function $f$, where $p_0 = \frac{1}{vol(P)}\int_P x\,\text{d}x$ denotes the barycenter of $P$.
+  \bigskip
+  
+  Let $f$ be a linear function $f:\mathbb{R}^d\longrightarrow\mathbb{R}^n$.
+  Then $f$ can be expressed as $f(x)=Ax+b$ where $A$ is a $nxd$ matrix and $b$ is a $nx1$ vector. Then:
+  
+  \begin{equation*}
+       \int_P f(x)\,\text{d}x= \int_P Ax+b\,\text{d}x = A\int_P x\,\text{d}x + b\int_P \,\text{d}x =  (vol P) \bigg(\frac{A}{vol(P)}\int_P x\,\text{d}x + b\bigg) = vol(P)*f(p_0)
+  \end{equation*}
+  where $p_0 = \frac{1}{vol(P)}\int_P x\,\text{d}x$ is the barycenter of $P$.
+  \bigskip
+\item Complete the proof of Theorem 9.6 in Ziegler's \emph{Lectures on Polytopes}.
+  
+\end{enumerate}
+\begin{defi}
+Let $\gamma$ be a section. $\gamma$ is a tight section if and only if $\mathrm{Im}\gamma=\bigcup_{F\in\mathcal{F}}F$ for some $\mathcal{F}\subseteq\mathcal{L}\left(P\right)$.
```

Comment:
"$\gamma$ is tight if and only if..." (avoid repeat of "section")
or:
"A section $\gamma: \pi(P)\to P$ is tight if and only if..." (avoid repeat of $\gamma$ and section, and clarify domain and range of $\gamma$)